tutorials/Anatomy of a Device.ipynb

```diff
@@ -25,8 +25,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Runs EPICS IOC(s) with simulated hardware in leiu of actual motors, detectors.\n",
-    "!supervisor/start_supervisor.sh"
+    "# Monitor EPICS IOC(s) with simulated hardware in leiu of actual motors, detectors.\n",
+    "!supervisor/monitor_supervisor.sh"
```

Comment:
I think we can delete `monitor_supervisor.sh` and simply put the command directly here:

```
!supervisorctl -c supervisor/supervisor.conf status
```

That makes it easy to run a different command (e.g. `restart all` in place of `status`) as needed. It also makes it easier for an intermediate/advanced user to understand what is happening here.